src/components/tooltip/examples/tooltip-composite.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component, h, Prop, State } from '@stencil/core';
```

Comment:
@LucyChyzhova I squashed your fixups, and added a few commits. If you're happy with my changes and additions, I'm happy to squash and merge 🙂